coverage/cmdline.py

```diff
@@ -24,7 +25,8 @@
 from coverage.control import DEFAULT_DATAFILE
 from coverage.data import combinable_files, debug_data_file
 from coverage.debug import info_header, short_stack, write_formatted_info
-from coverage.exceptions import _BaseCoverageException, _ExceptionDuringRun, NoSource
+from coverage.exceptions import _BaseCoverageException, _ExceptionDuringRun, NoSource, \
+    NoDataFilesFoundError
```

Comment:
Please use parentheses instead of backslashes:
```suggestion
from coverage.exceptions import (
    _BaseCoverageException, _ExceptionDuringRun, NoSource, NoDataFilesFoundError,
)
```